### ISSUE TITLE
browse_directories's directory view's cursor resets to first directory in list along with "track in" view, but does not reset position in directory view

### ISSUE THREAD

**Issue by PythonTryHard** [clangen/musikcube#378]
**Version affected:**
- `0.93.1` on Windows 10 2004 build 19041.572

**Steps to reproduce:**
1. Switch to directory view with `navigate_library_browse_directories` key
2. Navigate to any folder in the directory view.
3. Switch to library view with `navigate_library_tracks` key
4. Switch back to directory with `navigate_library_browse_directories` key

**Initial state:**
![image](https://user-images.githubusercontent.com/31789326/97781524-a3285e00-1bbe-11eb-84d0-2597e45d8003.png)

**Expected behavior:**
Back to initial state or at least have the previously selected directory being "on top"
![image](https://user-images.githubusercontent.com/31789326/97781549-d539c000-1bbe-11eb-8751-8105ce962956.png)

**Actual behavior:**
The cursor is at the first directory in the list. By selecting the directory below then back up, or straight up hitting Up at this point will cause the view to snap back to the first directory
![image](https://user-images.githubusercontent.com/31789326/97781586-116d2080-1bbf-11eb-845a-32831379a276.png)

**Comment by clangen** [comment]
This no longer seems to be a problem. I just tested on the `master` branch on both Windows and Linux and it seems fine. This was likely incidentally fixed during some window/layout focus-behavior refactors that went into remote library support, but may have been fixed more recently. At any rate, I'm flagging it as fixed so you can give it a test when we release a new build.

**Comment by PythonTryHard** [comment]
Tested with d79dac8a freshly built against `Boost 1.75.0 x64` and unfortunately I'm still seeing it. Here's a video of reproduction on my end:

https://user-images.githubusercontent.com/31789326/114665815-ac9a2280-9d27-11eb-90f0-59ad9d39aba7.mp4

I switched to directory view with `d` then hit `End` to move to the end of the directory listing. I then hit `t` to move back to track view followed `d` to get back to directory view. Notice on the second return to directory view the `directory` view still shows the end of the directory listing while `tracks in F:\Music` shows the content of the first entry.
